frontend/src/organisms/Actions.js

```diff
@@ -0,0 +1,115 @@
+import types from './types';
```

Comment:
tohle nebude organismus, ne? je to nějaká lgika, nic pro vizualizaci

Reply:
přesunuto do utils